packages/docusaurus-plugin-content-docs/src/sidebars.ts

```diff
@@ -210,6 +227,7 @@ function normalizeItem(item: SidebarItemJSON): UnprocessedSidebarItem[] {
       return [
         {
           collapsed: DefaultCategoryCollapsedValue,
+          collapsible: DefaultCategoryCollapsibleValue,
```

Comment:
replace plugin option default value?

Reply:
also, we could throw an error if `!collapsible && collapsed` as the user probably made a mistake